part-one-answer.js

```diff
@@ -0,0 +1,17 @@
+function showresult(val){
+    var n1=parseFloat(document.getElementById('num1').value);
+    var n2=parseFloat(document.getElementById('num2').value);
+    var r=n1+n2;
```

Comment:
More descriptive naming may have saved you the issue below. While it's easy to keep track of what `n1` is when you have 3 variables, it becomes really difficult with hundreds or thousands of them as you'll find in most code bases.

```JavaScript
var num1 = parseFloat(document.getElementById('num1').value)
var num2 = parseFloat(document.getElementById('num2').value)
var result = n1 + n2
```